src/src/DataStructs/PluginStats.cpp

```diff
@@ -120,14 +120,16 @@ float PluginStats::getSampleExtreme(PluginStatsBuffer_t::index_t lastNrSamples,
   return res;
 }
 
-float PluginStats::getSample(PluginStatsBuffer_t::index_t lastNrSamples) const
+float PluginStats::getSample(int& lastNrSamples) const
 {
-  if ((_samples.size() == 0) || (_samples.size() < lastNrSamples)) { return _errorValue; }
+  if ((_samples.size() == 0) || (_samples.size() < abs(lastNrSamples))) { return _errorValue; }
 
   PluginStatsBuffer_t::index_t i = 0;
 
-  if (lastNrSamples < _samples.size()) {
+  if (lastNrSamples >= 0) {
     i = _samples.size() - lastNrSamples;
```

Comment:
with the index being 0, you will get a crash here.

Reply:
This has already been fixed in the next commits, sorry.